odl_multigrid/graphics.py

```diff
@@ -102,6 +102,48 @@ def show(data, eps=0.0):
     # TODO: set aspect from physical sizes
     ax.set_aspect('auto', 'box')
 
+def show_all(data, low = None, high = None):
+    """Show all reconstructed resolutions in a single image. For multiple ROIs"""
```

Comment:
Maybe a word on what ``data`` is expected to be?

Reply:
Yes. Added info in a2a42d3 in the form of

- `data: list of reconstructions, with background given as data[0] and
              subsequent ROI reconstructions given as data[1], data[2], ...`
- `low: lower bound of dynamic range. If 'None' set to min(data)`
- `high: upper bound of dynamic range. If 'None' set to max(data)`